### PR TITLE
K8SPG-129 document custom configuration

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: source/options.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+You can add options by editing the configuration section of the
+``deploy/cr.yaml``. Here is an example:
+
+.. code:: yaml
+
+   spec:
+     ...
+     pgPrimary:
+       ...
+         customconfig: |
```
Comment (spron-in) [inline source/options.rst:28]: in other operators we have `configuration` not `customconfig`. Is it something we inherited @hors  ?

File: source/options.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+See :ref:`operator.custom-resource-options` for more details.
+
+.. _operator-configmaps-cm:
+
+Use a ConfigMap
+---------------
+
+You can use a configmap and the cluster restart to reset configuration
+options. A configmap allows Kubernetes to pass or update configuration
+data inside a containerized application.
```
Comment (spron-in) [inline source/options.rst:42]:
```suggestion
You can use a ConfigMap and the cluster restart to reset configuration
options. A ConfigMap allows Kubernetes to pass or update configuration
data inside a containerized application.
```

File: source/options.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+.. code:: bash
+
+   kubectl describe configmaps cluster1-pgo
+
+.. _operator-configmaps-restart:
+
+Make changed options visible to PostgreSQL
+------------------------------------------
+
+Do not forget to restart the cluster to ensure it has updated the configuration
```
Comment (spron-in) [inline source/options.rst:89]: I don't understand this section. 

1. should I restart if I create configmap or if I apply config with cr.yaml? Both?
2. How do I restart? 
3. Aren't we restarting automatically?

File: source/options.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+(see details on how to connect in the `Install Percona Distribution for PostgreSQL on Kubernetes <kubernetes.html>`_ page).
```
Comment (spron-in) [inline source/options.rst:90]: I don't think this is relevant and you wanted to have a link to how to restart here?

File: source/options.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+Changing PostgreSQL Options
+===========================
+
+You may require a configuration change for your application. PostgreSQL
+allows the option to configure the database with a configuration file.
+You can pass the `PostgreSQL configuration options <https://www.postgresql.org/docs/9.3/config-setting.html>`__
+in one of the following ways:
+
+* edit the ``deploy/cr.yaml`` file,
+* use a ConfigMap.
```
Comment (spron-in) [inline source/options.rst:12]: If I use this, will my PostgreSQL configuration be rewritten or appended to existing ones? We need to mention this.

**Review by spron-in — CHANGES_REQUESTED** [review]

**Review by hors — CHANGES_REQUESTED** [review]

File: source/options.rst [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+To change options for an existing cluster, you can do the same but put options
+in a ``postgres-ha.yaml`` file directly, without the ``bootstrap`` section.
+
+.. _operator-configmaps-create:
+
+Creating a cluster with custom options
+--------------------------------------
+
+For example, you can create a cluster with a custom ``max_connections`` option
+in a ``proxysql.conf`` configuration file using the following ``postgres-ha.yaml``
```
Comment (spron-in) [inline source/options.rst:48]:
```suggestion
in a ``postgresql.conf`` configuration file using the following ``postgres-ha.yaml``
```

File: source/options.rst [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+         customconfig: "cluster1-custom-config"
+
+Now you can create the cluster following the :ref:`regular installation instructions<_operator-install>`.
+
+.. _operator-configmaps-change:
+
+Modifying options for the existing cluster
+------------------------------------------
+
+For example, you can change ``max_connections`` option in a ``proxysql.conf``
```
Comment (spron-in) [inline source/options.rst:102]:
```suggestion
For example, you can change ``max_connections`` option in a ``postgresql.conf``
```

**Review by spron-in — CHANGES_REQUESTED** [review]
Few more comments + my previous comments are not resolved.

**Review by hors — APPROVED** [review]

**Pull request merged**